packages/lesswrong/components/community/modules/SearchResultsMap.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import { Components, registerComponent } from '../../../lib/vulcan-lib';
+import { createStyles } from '@material-ui/core/styles';
+import ReactMapGL, { Marker } from 'react-map-gl';
+import { Helmet } from 'react-helmet'
+import { forumTypeSetting } from '../../../lib/instanceSettings';
+import { mapboxAPIKeySetting } from '../../../lib/publicSettings';
+import { connectHits } from 'react-instantsearch-dom';
+import PersonIcon from '@material-ui/icons/PersonPin';
+import { userGetProfileUrl } from '../../../lib/collections/users/helpers';
+
+const styles = createStyles((theme: ThemeType): JssStyles => ({
+  root: {
+    width: "100%",
+    height: 440,
+  },
+  icon: {
+    height: 20,
+    width: 20,
+    fill: theme.palette.individual,
+    opacity: 0.8,
+    cursor: 'pointer'
+  },
+  popupAddress: {
+    ...theme.typography.commentStyle,
+    color: theme.palette.grey[600],
+    fontSize: 12,
+    fontStyle: 'italic',
+  },
+  popupBio: {
+    ...theme.typography.commentStyle,
+    color: theme.palette.grey[800],
+    fontSize: 14,
+    lineHeight: '1.8em',
+    whiteSpace: 'pre-wrap',
+    display: '-webkit-box',
+    "-webkit-line-clamp": 3,
+    "-webkit-box-orient": 'vertical',
+    overflow: 'hidden',
+    marginTop: 12,
+  }
+}))
+
+const defaultCenter = {lat: 39.5, lng: -43.636047}
+
+const SearchResultsMap = ({center = defaultCenter, zoom = 2, hits, classes}: {
```

Comment:
I started by using the existing `CommunityMap` component, and by the end decided to just make my own. This way I could insert it into `<InstantSearch>` (and bonus, no more negative top margin to account for!). It's styled very similarly to the existing `CommunityMap`. Seems like there are fancier things we can do with Algolia and geosearching, but I figured I should stop here for the MVP and improve on it later.